test/fw/bin/pbs_benchpress

```diff
@@ -65,13 +65,16 @@ def trap_exceptions(etype, value, tb):
         pass
     else:
         sys.__excepthook__(etype, value, tb)
+
+
```

Comment:
I think the single blank line rule is only valid for functions within a class: http://legacy.python.org/dev/peps/pep-0008/#blank-lines